Dalamud/Interface/ImGuiFileDialog/DriveListLoader.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Dalamud.Interface.ImGuiFileDialog;
+
+/// <summary>
+/// A drive list loader. Thread-safety guaranteed.
+/// </summary>
+public class DriveListLoader
```

Comment:
Why is this public? I feel like this should be an implementation detail. Otherwise, looks good.

Reply:
Updated